quinn-proto/src/congestion/new_reno.rs

```diff
@@ -25,21 +27,37 @@ impl NewReno {
             ssthresh: u64::max_value(),
             recovery_start_time: now,
             config,
+            bytes_acked: 0,
         }
     }
 }
 
 impl Controller for NewReno {
-    fn on_ack(&mut self, _now: Instant, sent: Instant, bytes: u64, congestion_blocked: bool) {
-        if !congestion_blocked || sent <= self.recovery_start_time {
+    fn on_ack(&mut self, _now: Instant, sent: Instant, bytes: u64, app_limited: bool) {
+        if app_limited || sent <= self.recovery_start_time {
             return;
         }
+
         if self.window < self.ssthresh {
             // Slow start
             self.window += bytes;
+
+            if self.window >= self.ssthresh {
+                // Exiting slow start
+                // Initialize `bytes_acked` for congestion avoidance
+                self.bytes_acked = self.window - self.ssthresh
```

Comment:
What's this logic based on? Also, style nit:
```suggestion
                self.bytes_acked = self.window - self.ssthresh;
```